main/webapp/modules/core/scripts/reconciliation/recon-dialog.js

```diff
@@ -31,11 +31,12 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
  */
 
-function ReconDialog(column, types) {
+function ReconDialog(column,previousServiceName) {
```

Comment:
Rather than using the name of a service to remember the previous selection, I think it would be better to use the endpoint. We already guarantee that endpoints are unique, whereas there could be two different services with the same name.

Also, mind the indentation: function arguments are normally not just separated by a comma, but also a space after the comma.